merlin/examples/workflows/hello/hello.yaml

```diff
@@ -10,15 +10,28 @@ global.parameters:
         values : ["world","mundo"]
         label  : WORLD.%%
 
+user:
```

Comment:
If we're changing this, we need to make sure the tutorial aligns with it (since the tutorial uses hello_world). I suggest we don't modify this file and keep it simple

Reply:
Okay sounds good. I removed it from hello.yaml example